op-e2e/e2eutils/blobs.go

```diff
@@ -0,0 +1,48 @@
+package e2eutils
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+// BlobsStore is a simple in-memory store of blobs, for testing purposes
+type BlobsStore struct {
+	blobs map[uint64]map[common.Hash]*eth.Blob
```

Comment:
Why don't we store by block hash? If we're ever gonna use the `BlobStore` in a reorg scenario, this would come in handy. From what I can tell, it would work if we just change `StoreBlob`'s first parameter to a hash.

Reply:
Will change it to hash :+1: